### DISCUSSION TITLE
Idea: Add custom buttons to frigate card

### DISCUSSION THREAD

**Discussion by slobdellreachout** [blakeblackshear/frigate#14235] [Ideas]
Love Frigate, was thinking it would be cool to be able to add custom buttons to the live view, or a button to access a popup of custom buttons with any HA action. So if I'm watching a live stream of my front yard and I want to turn the lights on out there, I could do it right from the live stream instead of having to go to a different tab, turn them on, and come back. Not sure how feasible that is. Thanks!

**Comment by hawkeye217** [comment]
This is the repo for Frigate itself, the card is maintained by another author and the repo is here: https://github.com/dermotduffy/frigate-hass-card

**Reply by slobdellreachout** [reply]
Oh thanks!
